### PR TITLE
create text, list, nav component

### PR DESCRIPTION
**Pull request by git-0r**
# Utility classes for text
 - heading
 - gray text
 - text alignment
text1

# List component
 - with checkbox & radio
 - stacked
list1
list2

# Navigation
nav1

### CODE REVIEW

**Review by Sahil25061999 — APPROVED** [review]
Looks good to me  didnt find any bugs. it can be merged.
Just a suggestion ,  I saw you are using margin: 0 in lot of places, you could just use reset css for that.

**Comment by git-0r** [comment]
> Looks good to me didnt find any bugs. it can be merged. Just a suggestion , I saw you are using margin: 0 in lot of places, you could just use reset css for that.

i want to keep the default spacing, so i am applying reset only on some elements like text & heading. And using margin: 0 on smaller components like badges

**Review by yaswanthmyneni — APPROVED** [review]
looks good to me. You can merge after changes.

File: css/comp_lib_styles/components/list.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.styled-list {
+  list-style: none;
+  padding-left: 1rem;
+}
+
+/* ---------apply margin on all list items-------- */
+
+.styled-list > * {
+  margin: 0.5rem;
```
Comment (yaswanthmyneni) [inline css/comp_lib_styles/components/list.css:9]: changes: use px here.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes.

**Pull request merged**